doc/storageprofile.md

```diff
@@ -0,0 +1,198 @@
+# Storage Profiles
+
+## Introduction
+
+Storage Profile is the resource that  serves the information about recommended parameters for PVC - right now the accessMode and volumeMode.
+This can be used by CDI controllers when creating a PVC for DV. That way the DataVolume can be simplified and if the properties are missing,
+defaults can be applied from the StorageProfile.
+
+CDI provides a collection of Storage Profiles with default recommended values for some well known backends. 
+If the storage provisioner defined in storage class does not have defaults configured in CDI the resulting StorageProfile 
+has empty `claimPropertySets`.
+
+Storage profile maps 1:1 to StorageClass.
```

Comment:
Might be worth stating explicitly here that CDI automatically creates these objects -- one per storage class that exists in the cluster.

Reply:
done